protocol/persistence_quoted_message_test.go

```diff
@@ -198,3 +199,39 @@ func (s *TestMessengerPrepareMessageSuite) Test_WHEN_QuotedMessageDoesNotContain
 
 	s.Require().Equal(json.RawMessage(nil), retrievedMessages[0].QuotedMessage.AlbumImages)
 }
+
+func (s *TestMessengerPrepareMessageSuite) Test_WHEN_MessageContainsImage_THEN_PreparedMessageByIDContainsLink() {
+	// GIVEN: message with image saved to DB
+	message := s.generateTextMessage("id-1", "1", 1, "")
+	message.ContentType = protobuf.ChatMessage_IMAGE
+	message.Payload = &protobuf.ChatMessage_Image{
+		Image: &protobuf.ImageMessage{
+			Format:  1,
+			Payload: RandomBytes(100),
+		},
+	}
+
+	err := s.m.SaveMessages([]*common.Message{message})
+	s.Require().NoError(err)
+
+	messages := []*common.Message{message}
+	err = s.p.SaveMessages(messages)
+	s.Require().NoError(err)
```

Comment:
Are you calling the same thing twice? 🤔

Reply:
> Are you calling the same thing twice? 🤔
yes, let me check if calling only s.m.SaveMessages works

Reply:
> yes, let me check if calling only s.m.SaveMessages works

I've removed the call to `Persistence.SaveMessage` because `Messenger.SaveMessage` calls the Persistence method. Test works fine